.pre-commit-config.yaml

```diff
@@ -1,77 +1,29 @@
-#
-# Run pre-commit hooks. You can run them without installing
-#  the hook with
-#
-# $ pre-commit run --all-files
-#
-# See https://pre-commit.com for more information
-# See https://pre-commit.com/hooks.html for more hooks
 repos:
--   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.3.0
-    hooks:
-    -   id: trailing-whitespace
-    -   id: end-of-file-fixer
-    -   id: check-yaml
-        args: [--allow-multiple-documents]
-    -   id: check-added-large-files
-        args:
-        - "--maxkb=4000"
-- repo: https://github.com/myint/autoflake
-  rev: v1.4
-  hooks:
-    - id: autoflake
-      args:
-        - --in-place
-        - --remove-unused-variables
-        - --remove-all-unused-imports
--   repo: https://github.com/psf/black
-    rev: 22.6.0
-    hooks:
-    -   id: black
-- repo: https://github.com/pycqa/isort
-  rev: 5.10.1
-  hooks:
-    - id: isort
-      name: isort (python)
-      # Use black profile for isort to avoid conflicts
-      #   see https://github.com/PyCQA/isort/issues/1518
-      args: ["--profile", "black"]
-    - id: isort
-      name: isort (cython)
-      types: [cython]
-    - id: isort
-      name: isort (pyi)
-      types: [pyi]
-- repo: https://gitlab.com/pycqa/flake8
-  rev: 3.9.2
+- repo: https://github.com/teamdigitale/dati-semantic-cookiecutter
+  rev: 931e0529c8839f6fa8c1ae315839ba7c3060c5f2
   hooks:
-  - id: flake8
-- repo: https://github.com/PyCQA/bandit
-  rev: 1.7.4
-  hooks:
-    - id: bandit
-      name: bandit
-      args: ["-c", ".bandit.yaml"]
-      description: 'Bandit is a tool for finding common security issues in Python code'
-      entry: bandit
-      language: python
-      language_version: python3
-      types: [python]
-- repo: https://github.com/Lucas-C/pre-commit-hooks-safety
-  rev: v1.3.0
-  hooks:
-    - id: python-safety-dependencies-check
+  - id: check-repo-structure
+    args: ["assets/controlled-vocabularies/", "assets/ontologies/", "assets/schemas/"]
+  - id: check-filename-format
+    args: ["assets/controlled-vocabularies/", "assets/ontologies/", "assets/schemas/"]
+  - id: check-filenames-match-uri
+    args: ["assets/controlled-vocabularies/", "assets/ontologies/", "assets/schemas/"]
+  - id: check-filenames-match-directories
+    args: ["assets/controlled-vocabularies/", "assets/ontologies/", "assets/schemas/"]
+  - id: check-supported-files
+    args: ["assets/controlled-vocabularies/", "assets/ontologies/", "assets/schemas/"]
+  - id: check-versioning-pattern
+    args: ["assets/controlled-vocabularies/", "assets/ontologies/", "assets/schemas/"]
```

Comment:
If the files that execute these checks are published in the repository, they will eventually diverge between various agencies.

These hooks should be installe from a central repository (e.g., added to json-semantic-playground or to a `dati-semantic-hooks` repository)